### ISSUE TITLE
[Feature Request]: Refactor Translation Keys

### ISSUE THREAD

**Issue by pxlrbt** [laravel-idea/plugin#1038]
### Feature Description

It would be nice to be able to refactor translation keys. Sometimes I want to move some of them around like from `app.edit` to `app.pages.edit`, but then I have to search for this Translation-Key everywhere and hope I don't accidentally change a view name or similar.

**Comment by studiobts** [comment]
I agree, I have over 7K translation keys, each one in 6 languages. Moving a key or a "block" of keys is always a panic. A refactor helper would be super useful!

Any chance to get it implemented?

**Comment by gorbunov** [comment]
> Any chance to get it implemented?

You may check plugin dedicated to [translations support](https://locale-sphere.com) it may help you operating larger translations codebase.
